main.go

```diff
@@ -152,9 +183,15 @@ func main() {
 
 	done := timeline.Start()
 
+	if MetricsEndpointBasicAuthEnabled() {
+		http.Handle("/metrics", BasicAuth(promhttp.Handler()))
+	} else {
+		http.Handle("/metrics", promhttp.Handler())
+	}
+
 	// Self monitoring
-	http.Handle("/metrics", promhttp.Handler())
-	_ = http.ListenAndServe(fmt.Sprintf(":%d", listenPort), nil)
+	err = http.ListenAndServe(fmt.Sprintf(":%d", listenPort), nil)
+	fmt.Printf("%s", err.Error())
```

Comment:
Make this printout conditional i.e. only if err != nil

Reply:
Added if statement